lib/gltfPipeline.js

```diff
@@ -67,6 +71,10 @@ function processJSONWithExtras(gltfWithExtras, options) {
     var stats = new OptimizationStatistics();
     addDefaults(gltfWithExtras, options);
     removeUnused(gltfWithExtras, stats);
+    mergeDuplicateShaders(gltfWithExtras);
```

Comment:
_As of now_, is there any reason to also run these stages at the end of the pipeline, e.g., because other stages blindly add new shaders, programs, etc.?